app/controllers/submit.js

```diff
@@ -247,8 +256,8 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             this.set('node.title', this.get('nodeTitle'));
             let node = this.get('node');
             node.save();
+            this.notifyPropertyChange('node.title');
```

Comment:
Given that we call `this.set()` on title above, can you please comment on the reason for needing `notifyPropertyChange` here? I would expect observers to pick up on the property change without this line, since we appear to be using normal setting mechanisms.